### ISSUE TITLE
remaining Anna Ho comments

### ISSUE THREAD

**Issue by davidwhogg** [andycasey/AnniesLasso#57]
- [x] Figure order (Figures are not referred to in order)?
- [x] Units on plots - inconsistent in that sometimes there, sometimes not
- [x] Anna suggests stating we do not create ground truth we only propagate the training set (see email for more details).

**Comment by andycasey** [comment]
Regarding the "units on plots" item (cc @annayqho):

I only use "dex" units for abundances when they are referring to a _difference_ in abundances, not respect to Solar. For example in `fig:label-recovery-snr`, units are only used in abundances because it is showing the difference with respect to the high S/N combined spectrum. As far as I can see, units are consistently used elsewhere.

**Comment by annayqho** [comment]
Got it! Makes sense, thanks.

**Comment by andycasey** [comment]
I think this can be closed, but I will leave it open for @davidwhogg to close.
